packages/workspace-tools/src/generators/extend-interface/extend-interface.ts

```diff
@@ -80,7 +82,9 @@ async function createImportDeclaration(
 
             return statement.exportClause.elements.filter(ts.isExportSpecifier);
         })
-        .map(element => element.name.escapedText.toString())
+        .map(element => {
+            return element.name.escapedText.toString();
```

Comment:
Is this diff intentional?

Reply:
There were issues with this executor, that I had to debug. So left behind, sorry about that.